multiqc/modules/hicpro/hicpro.py

```diff
@@ -0,0 +1,386 @@
+             'hidden': True
+         }
+
+         headers['percent_paired_reads'] = {
+             'title': '% Reported Read Pairs',
+             'description': 'Percentage of paired reads (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': False
+         }
+
+         headers['valid_interaction'] = {
+             'title': '{} Valid Pairs'.format(config.read_count_prefix),
+             'description': 'Number of valid pairs ({})'.format(config.read_count_desc),
+             'min': 0,
+             'scale': 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key': 'read_count',
+             'hidden': True
+         }
+         
+         headers['percent_valid'] = {
+             'title': '% Valid Pairs',
+             'description': 'Percentage of valid pairs over reported ones (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': True
+         }
+         
+         headers['valid_interaction_rmdup'] = {
+             'title': '{} Valid Pairs Unique'.format(config.read_count_prefix),
+             'description': 'Number of valid pairs after duplicates removal ({})'.format(config.read_count_desc),
+             'min': 0,
+             'scale': 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key': 'read_count',
+             'hidden': True
+         }
+         
+         headers['percent_valid_interaction_rmdup'] = {
+             'title': '% Valid Pairs Unique',
+             'description': 'Percent of valid pairs over reported ones after duplicates removal (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': False
+         }
+     
+         self.general_stats_addcols(self.hicpro_data, headers, 'HiC-Pro')
+
+
+    def hicpro_mapping_chart (self):
+        """ Generate the HiC-Pro Aligned reads plot """
+
+        # Specify the order of the different possible categories
+        keys = [OrderedDict(), OrderedDict()]
+        keys[0]['Full_Alignments_Read']   = { 'color': '#005ce6', 'name': 'Full reads Alignments' }
+        keys[0]['Trimmed_Alignments_Read'] = { 'color': '#3385ff', 'name': 'Trimmed reads Alignments' }
+        keys[0]['Failed_To_Align_Read']     = { 'color': '#a9a2a2', 'name': 'Failed To Align' }
+        keys[1]['Full_Alignments_Read']   = { 'color': '#005ce6', 'name': 'Full reads Alignments' }
+        keys[1]['Trimmed_Alignments_Read'] = { 'color': '#3385ff', 'name': 'Trimmed reads Alignments' }
+        keys[1]['Failed_To_Align_Read']     = { 'color': '#a9a2a2', 'name': 'Failed To Align' }
+                           
+        data = [{},{}]
+        for s_name in self.hicpro_data:
+            for r in [1,2]:
+                data[r-1][s_name] = {
```

Comment:
If you want to, you could also add in `R1` and `R2` to the sample name if you want to. It could make things a little clearer when exporting the plot as a stand-alone for example.

```python
data[r-1]['{} R{}'.format(s_name, r)]
```

Reply:
thanks